<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.30.2" />


<title>About - Personal Website for Ted Laderas, PhD</title>
<meta property="og:title" content="About - Personal Website for Ted Laderas, PhD">



  







<link rel="stylesheet" href="/css/fonts.css" media="all">
<link rel="stylesheet" href="/css/main.css" media="all">



  </head>
  <body>
    <div class="wrapper">
      <header class="header">
        <nav class="nav">
  <a href="/" class="nav-logo">
    <img src="/images/logo.png"
         width="50"
         height="50"
         alt="Logo">
  </a>

  <ul class="nav-links">
    
    <li><a href="/about/">About</a></li>
    
    <li><a href="/cv/">CV</a></li>
    
    <li><a href="/teaching/">Teaching</a></li>
    
    <li><a href="http://github.com/laderast">GitHub</a></li>
    
    <li><a href="https://twitter.com/tladeras">Twitter</a></li>
    
  </ul>
</nav>

      </header>


<main class="content" role="main">

  <article class="article">
    

    <h1 class="article-title">About</h1>

    

    <div class="article-content">
      

<p>I am an Assistant Professor in the Bioinformatics and Computational Biology track in the OHSU Department of
Medical Informatics and Clinical Epidemiology (<a href="http://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/clinical-departments/dmice/educational-programs/dmice-programs/computational-biology.cfm">BCB/DMICE</a>) and a member of the <a href="http://www.ohsu.edu/xd/health/services/cancer/">OHSU Knight Cancer Institute</a>. My research focus is on the Systems Biology of Complex Diseases. I use integrative modeling approaches across OMICs types to achieve this.</p>

<p>Over twelve years of bioinformatics experience, I have developed methods for low-level preprocessing and high-level analysis of many OMICs types, including expression (especially alternative splicing), phosphoproteomics, proteomics, and genomics. My experience in functional genomics has led me to use models such as Graphical Models and Networks to integrate these many OMICs types into building a unified picture of integrated systems-level dysregulation in complex diseases such as alcoholism and cancer.</p>

<h2 id="research-interests"><strong>Research Interests</strong></h2>

<ul>
<li>Phenotypic Heterogeneity of Cancer and Measurement using Flow Cytometry Methods</li>
<li>Systems Biology of Cancer, especially oncogenic collaboration</li>
<li>Prioritization of Combination Therapies in the treatment of cancer</li>
<li>Integrating multi-OMICs data to understand dysregulation in cancer</li>
<li>Modeling Disease using Graphical Models and Agent-Based Models</li>
<li>Interactive visualization of Data</li>
</ul>

<h3 id="software-development"><strong>Software Development:</strong></h3>

<ul>
<li><a href="https://github.com/laderast/flowDashboard">flowDashboard</a>. Shiny modules for building interactive and comparative dashboards for flow cytometry data. See a demo here: <a href="https://tladeras.shinyapps.io/sampleFlowDashboard/">https://tladeras.shinyapps.io/sampleFlowDashboard/</a> <a href="https://zenodo.org/badge/latestdoi/72794399"><img src="https://zenodo.org/badge/72794399.svg" alt="DOI" /></a></li>
<li><a href="https://github.com/laderast/surrogateMutation">surrogateMutation</a> - a package for detecting oncogenic collaboration in somatic mutation and copy nubmer data. <a href="https://zenodo.org/badge/latestdoi/34889234"><img src="https://zenodo.org/badge/34889234.svg" alt="DOI" /></a></li>
<li><a href="https://github.com/laderast/surrogateShiny">surrogateShiny</a> - an R/Shiny Framework for exploring oncogenic collaboration in breast cancer cell lines. See a demo here: <a href="https://tladeras.shinyapps.io/surrogateShiny/">https://tladeras.shinyapps.io/surrogateShiny/</a> <a href="https://zenodo.org/badge/latestdoi/26290300"><img src="https://zenodo.org/badge/26290300.svg" alt="DOI" /></a></li>
<li><a href="https://github.com/laderast/surrogateMutation">Consense</a> - a package for comparing clustering methods. <a href="https://zenodo.org/badge/latestdoi/34889209"><img src="https://zenodo.org/badge/34889209.svg" alt="DOI" /></a></li>
<li><a href="https://github.com/laderast/ExonModelStrain">ExonModelStrain</a> - a package for detecting alternative exon usage in the Affymetrix Exon Array. <a href="https://zenodo.org/badge/latestdoi/34889191"><img src="https://zenodo.org/badge/34889191.svg" alt="DOI" /></a></li>
<li>TandTRAQ - Perl Script for merging iTRAQ and XTandem Results.</li>
</ul>

<h3 id="selected-publications"><strong>Selected Publications:</strong></h3>

<ul>
<li><a href="http://journal.frontiersin.org/article/10.3389/fgene.2015.00341/abstract">A Network Based Model of Oncogenic Collaboration to Predict Drug Sensitivity.</a> <strong>Ted Laderas</strong>, Kemal Sonmez, and Laura Heiser. Frontiers in Genetics. 2015.</li>
<li><a href="http://www.researchgate.net/publication/281769232_Between_pathways_and_networks_lies_context_Implications_for_precision_medicine">Between Networks and Pathways Lies Context.</a> <strong>Ted Laderas</strong>, Guanming Wu, and Shannon McWeeney. Science Progress. 2015.</li>
<li><a href="http://www.nature.com/nm/journal/vaop/ncurrent/full/nm.3967.html">The Consensus Molecular Subtypes of Colorectal Cancer</a>. Justin Guinney, Rodrigo Dienstmann, Xin Wang, Aurélien de Reyniès, Andreas Schlicker, Charlotte Soneson, Laetitia Marisa, Paul Roepman, Gift Nyamundanda, Paolo Angelino, Brian M. Bot, Jeffrey S. Morris, Iris Simon, Sarah Gerster, Evelyn Fessler, Felipe de Sousa e Melo, Edoardo Missiaglia, Hena Ramay, David Barras, Krisztian Homicsko, Dipen Maru, Ganiraju C. Manyam, Bradley Broom, Valerie Boige, <strong>Ted Laderas</strong>, Ramon Salazar, Joe W. Gray, Douglas Hanahan, Josep Tabernero, Rene Bernards, Stephen H. Friend, Pierre Laurent-Puig, Jan P. Medema, Anguraj Sadanandam, Lodewyk Wessels, Mauro Delorenzi, Scott Kopetz, Louis Vermeulen, and Sabine Tejpar. Nature Medicine. 2015.</li>
<li><a href="http://journal.frontiersin.org/article/10.3389/fnins.2011.00069/abstract">Computational detection of alternative exon usage.</a> <strong>Ted Laderas</strong>, Nicole Walter, Michael Mooney, Kristina Vartanian, Priscila Darakjian, Kari Buck, Chris Harrington, John Belknap, Robert Hitzemann, and Shannon McWeeney. Frontiers in Neurogenomics. 2011. Article 69. PMID 21625610. In this paper, Nicole Walter and I developed a computational framework for detecting alternative exon usage between strains of mice (B6/alcohol preferring and D2/wild type).</li>
<li><a href="http://bioinformatics.oxfordjournals.org/content/23/24/3394.full">TandTRAQ: An open-source tool for integrated protein identification and quantitation.</a> <strong>Ted Laderas</strong>, Cory Bystrom, Debra McMillen, Guang Fan and Shannon McWeeney.  Bioinformatics. 2007.</li>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17411399">A consensus framework for clustering microarray data.</a> <strong>Ted Laderas</strong> and Shannon McWeeney. OMICS: A Journal of Integrative Biology. 2007. 116-128.</li>
</ul>

<h3 id="other-interests"><strong>Other Interests</strong></h3>

<p>I have a parallel career as a composer and performing musician. I started taking cello lessons when I turned 30. I&rsquo;m a lifelong learner and I love to play with and accompany other musicians. For more information, please see <a href="http://15people.net">my artist webpage</a>.</p>

    </div>
  </article>

  


</main>

      <footer class="footer">
        <ul class="footer-links">
          <li>
            <a href="/index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
          </li>
          <li>
            <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="/images/hugo-logo.png" width="22" height="22"></a>
          </li>
        </ul>
      </footer>

    </div>
    

    

    
  </body>
</html>

